### PR TITLE
Feat/ handle inverse color for no stroke and configuration stroke

### PR DESCRIPTION
**Pull request by pairone**
[[中文版模板 / Chinese template](https://github.com/VisActor/vrender/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [x] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Release
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: packages/vrender-components/src/label/base.ts [hunk 1]
```diff
@@ -667,48 +663,114 @@ export abstract class LabelBase<T extends BaseLabelAttrs> extends AbstractCompon
...
-            fill: invertColor
-          });
-          break;
-        case 'similarSeries':
-          label.setAttributes({
-            fill: simialrColor
-          });
-          break;
-      }
+      const fillStrategy = option.fillStrategy ?? 'invertSeries';
```
Comment (xiaoluoHe) [inline packages/vrender-components/src/label/base.ts:666]: series 是 vchart 的概念。
从组件 or vgrammar 的角度来看，这样的配置选项，可能有点难以理解？

Comment (pairone) [reply packages/vrender-components/src/label/base.ts:666]: 可能改成baseMarkColor可能更符合这个语境

File: packages/vrender-components/src/label/base.ts [hunk 2]
```diff
@@ -667,48 +663,114 @@ export abstract class LabelBase<T extends BaseLabelAttrs> extends AbstractCompon
...
+              fill: simialrColor
+            });
+            break;
+        }
+
+        if (label.attribute.lineWidth === 0) {
+          continue;
+        }
+        switch (strokeStrategy) {
+          case 'null':
```
[the remark this answers is not in the document]
Comment (pairone) [reply packages/vrender-components/src/label/base.ts:694]: OK

File: packages/vrender-components/src/label/base.ts (hunk 1 above)
Comment (pairone) [reply packages/vrender-components/src/label/base.ts:666]: 我打算这样命名：

Comment (pairone) [reply packages/vrender-components/src/label/base.ts:666]: * - base（baseMark色），
* - invertBase（执行智能反色），
* - similarBase（智能反色的补色），
* - null（不执行智能反色，保持stroke设置的颜色）

File: packages/vrender-components/src/label/base.ts [hunk 3]
```diff
@@ -634,30 +634,26 @@ export abstract class LabelBase<T extends BaseLabelAttrs> extends AbstractCompon
...
-      // if (!isInside) {
-      //   continue;
-      // }
+      if (this.attribute.type === 'arc') {
+        if (this.attribute.position === 'inside') {
+          isInside = true;
+        } else {
+          isInside = false;
+        }
+      }
```
Comment (kkxxkk2019) [inline packages/vrender-components/src/label/base.ts:645]: 这个可以直接 `isInside = this.attribute.position === 'inside'`

Comment (pairone) [reply packages/vrender-components/src/label/base.ts:645]: get~

**Review by kkxxkk2019 — APPROVED** [review]

**Pull request merged**